utils/capabilities.c

```diff
@@ -105,9 +105,10 @@ gbSetCapabilties(void)
   }
 
   globalCapabilities = caps;
+  free_caps = false;
 
  out:
-  if (!globalCapabilities) {
+  if (free_caps) {
```

Comment:
I didn't get what extra are we solving here ?
previous statement was, if globalCapabilities is NULL, then free caps. globalCapabilities will be NULL in all fail exits of this function.

Reply:
Coverity doesn't understand global variable, and the flow that the given function gets called only once. So, it assumes global variable is set, and now the function is called again, and hence it traces all the paths where it can see leaks.

Yes, this is mostly dummy change functionality wise, but solves a coverity report.